plugins/woocommerce-blocks/bin/webpack-configs.js

```diff
@@ -38,6 +38,9 @@ let initialBundleAnalyzerPort = 8888;
 const getSharedPlugins = ( {
 	bundleAnalyzerReportTitle,
 	checkCircularDeps = true,
+	// Override dewp handlers if needed.
```

Comment:
@nielslange I'm referring to **D**ependency **E**xtraction **W**ebpack **P**lugin

but it's not clear so I'll just type the whole thing here, because probably not everyone knows what that is.

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/47105/commits/3c71600dc6d8cf66cd88451ec3d2957be3b249c5